pkg/omf/cli/omf.update.fish

```diff
@@ -15,14 +22,19 @@ function omf.update -a name
     set parent_path "pkg"
   end
 
-  for path in {$OMF_PATH,$OMF_CONFIG}/pkg/$name
+  set update_status error
+  for path in {$OMF_PATH,$OMF_CONFIG}/$parent_path/$name
     not test -e "$path/.git"; and continue
 
-    omf.repo.pull $path; and set return_success
-
-    set -q return_success;
-      and omf.bundle.install $path/bundle
+    omf.repo.pull $path
+    switch $status
+    case 304
+      set update_status uptodate
+    case 0
+      set update_status success
+      omf.bundle.install $path/bundle
+    end
   end
 
-  set -q return_success; and __omf.update.success "$name"
+  eval __omf.update.$update_status "$name"
```

Comment:
It is really needed to call a function dynamically/indirectly, through a variable's value.
There is of course the alternative of using $update_status as an argument to a `switch..case` block.
I don't intend to deteriorate coding standards, so tell me which way you find more appropriate.